NAMESPACE

```diff
@@ -62,6 +60,8 @@ export(updateAccountEnvVars)
 export(writeManifest)
 import(rlang)
 importFrom(lifecycle,deprecated)
+importFrom(snowflakeauth,snowflake_connection)
```

Comment:
Prefer `package::function()` rather than importing from that package into our namespace. 
https://r-pkgs.org/dependencies-in-practice.html#sec-dependencies-in-imports-r-code